content/docs/install.md

```diff
@@ -10,81 +10,71 @@ Odin is dead-simple to get started with!
 
 
-## Clone or download Odin
+## Clone or download Odin binaries
+You can either:
 
-Clone the repository (recommended, `git clone https://github.com/odin-lang/Odin`).
-Download [the latest release](https://github.com/odin-lang/Odin/releases/latest).
-Download [the latest nightly build](/docs/nightly/).
-
-**Note**: Cloning the repository is recommended in order to make [updating](#updating-the-compiler) easier.
-
-## Download and include necessary extras
-
-Odin supports x86-64/AMD64 on Windows, Linux and macOs, and ARM64 on macOS. Odin also relies on LLVM (for code generation) and an external linker.
+Clone the repository `git clone https://github.com/odin-lang/Odin` (recommended).
 
-#### For Windows...
-There's a couple prerequisites here. First, make sure you have Visual Studio installed; you have to compile Odin from source, and Odin also requires `link.exe` from VS anyway. The necesary LLVM components for Windows are included in the Odin repository.
+Or download the latest binaries and add them to your path:
 
-Now, it's time to build Odin and get started! Open the X64 Visual Studio command prompt ([if you don't typically use it, here's how to find it](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)) and navigate to the directory where you downloaded Odin. Run the `build.bat` file, and you should have a successfully built Odin compiler!
+Download [the latest release](https://github.com/odin-lang/Odin/releases/latest).
 
-To use Odin `link.exe` is required to be in the PATH of the callee as mentioned, this can either be achieved but calling Odin from the X64 Visual Studio command prompt or by calling the vcvarsall.bat (with x64 as an argument) script either in your shell or in your build script.
+Download [the latest nightly build](/docs/nightly/).
 
-#### For Mac and \*Nix...
+**Note**: Cloning the repository is recommended in order to make [updating](#updating-the-compiler) easier.
 
-For Linux, make sure you have `llvm` and `clang` installed through your package managers.
-For macOS, make sure you've installed the Xcode command-line tools (`xcode-select --install`), then install LLVM. If you use [Homebrew](https://brew.sh/), you can run `brew install llvm` to do this.
+### Support 
+Odin supports x86-64/AMD64 on Windows, Linux and macOS, and ARM64 on macOS. Odin also relies on LLVM (for code generation) and an external linker.
 
-Homebrew will not add LLVM to the PATH,
-- **Intel** : run `echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile` to add LLVM to your PATH.
-- **Apple Sillicon** : run `echo 'export PATH="/opt/homebrew/Cellar/llvm/14.0.6_1/bin:$PATH"' >> ~/.zshrc` to add LLVM to your PATH.
+### Requirements
+The following platform-specific steps are necessary:
 
-Then run `source ~/.bash_profile # or ~/.zshrc` to update your PATH variable in the current terminal session.
+- Windows
+    * Have Visual Studio installed (MSVC 2010 or later, for the linker)
```

Comment:
Perhaps mention that VS2019 and VS2022 have seen the most testing. VS2017 will likely work, but people installing Odin from code on Windows are likelier to have one of these latest two revisions installed?

Reply:
I think its a good notice, added it. I can change it latter if needed. I don't know if the project has a "minimum" supported version for the current target platforms and compilers, but I could test other VS versions just to make sure.